pkg/controller/multiclusterstatusaggregation/multiclusterStatusAggregation_controller.go

```diff
@@ -343,6 +343,7 @@ func (r *ReconcilePullModelAggregation) generateSummary(appSetClusterStatusMap m
 			Cluster:      cluster.clusterName,
 			SyncStatus:   appSetClusterStatusMap[appset][cluster].SyncStatus,
 			HealthStatus: appSetClusterStatusMap[appset][cluster].HealthStatus,
+			App:          appset.appset.Namespace + "/" + appset.appset.Name,
```

Comment:
```suggestion
			App:          appset.appset.Namespace + "/" + appset.appset.Name,
```
The app is in the same namespace as appset but the name of the app is different.
As it gathers the SyncStatus and HealthStatus in the code above, you will need to store the `app`